python/source/org/libreoffice/ide/eclipse/python/utils/TemplatesHelper.java

```diff
@@ -103,14 +103,14 @@ public static void copyTemplate(IUnoidlProject pProject, String pTemplateName,
      * @param pClazz the class from which to load the resource file
      * @param pDestPath the path in which the file should be copied, relatively to
      *          the project root.
-     * @param sameFileName the file to be copied to has same name as template or not 
+     * @param pSameFileName the file to be copied to has same name as template or not 
      * @param pArgs additional arguments to pass to the formatter
      */
     public static void copyTemplate(IProject pProject, String pTemplateName,
-        Class<?> pClazz, String pDestPath, boolean sameFileName, Object... pArgs) {
+        Class<?> pClazz, String pDestPath, boolean pSameFileName, Object... pArgs) {
```

Comment:
why the `p` prefix here?

Reply:
If we keep this rule, then the parameters of the methods start with `p`.